src/UndockedRegFreeWinRT/UndockedRegFreeWinRT/catalog.cpp

```diff
@@ -82,18 +82,32 @@ struct component
 
 static unordered_map<wstring, shared_ptr<component>> g_types;
 
-HRESULT WinRTLoadComponent(PCWSTR manifest_path)
+HRESULT WinRTLoadComponentFromFilePath(PCWSTR manifestPath)
 {
     ComPtr<IStream> fileStream;
-    ComPtr<IXmlReader> xmlReader;
+    RETURN_IF_FAILED(SHCreateStreamOnFileEx(manifestPath, STGM_READ, FILE_ATTRIBUTE_NORMAL, false, nullptr, &fileStream));
```

Comment:
This is a desktop-only API, which will limit where this library can be used. This isn't really intended for UWP scenarios, but is this the first time we would break this compat?
https://docs.microsoft.com/en-us/windows/win32/api/shlwapi/nf-shlwapi-shcreatestreamonfileex

Reply:
Discussed offline. We're not introducing a new dependency here and we've always been using xmllite that relies on this so we should be okay.